sdk/daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Upgrading.scala

```diff
@@ -158,6 +159,14 @@ object UpgradeError {
     override def message: String =
       s"The upgraded $origin has changed the order of its variants - any new variant must be added at the end of the enum."
   }
+
+  final case class DecreasingLfVersion(
```

Comment:
Would this be better named something `DowngradeAttempted`? Simply wondering if the current name is a little too generic?

Reply:
Naming is difficult and I navily not sure what would be the best.

- `DowngradeAttempted` sounds bad too me as there in my mind (up/down)grade is about the package version not the LF version.
- The `DecreasingLfVersion` extends `Upgrade.Error` so I did not find it so generic.